test/testenv/secretutil.go

```diff
@@ -123,7 +122,20 @@ func ModifySecretKey(deployment *Deployment, ns string, key string, value string
 		return false
 	}
 	//Modify data
-	data[key] = []byte(EncodeBase64(value))
-	modify := ModifySecretObject(deployment, data, ns)
+	data[key] = []byte(value)
+	modify := ModifySecretObject(deployment, data, ns, secretName)
 	return modify
 }
+
+//GetMountedKey Gets the key mounted on pod
+func GetMountedKey(deployment *Deployment, podName string, key string) string {
+	stdin := fmt.Sprintf("cat /mnt/splunk-secrets/%s", key)
```

Comment:
Nit: In the future you might want to check if the file for specific key exists or not at /mnt/splunk/<keyname> and also check if the Splunk Instance running in the pod has taken up the right value(if possible i.e hec_token might not be possible to validate but admin password, pass4Symmkeys might be possible)

Reply:
Would be adding splunk Instance verification for keys except for hex_token in further commits